core/storage/base_model/gae_models.py

```diff
@@ -61,6 +61,19 @@ class EntityNotFoundError(Exception):
         """Raised when no entity for a given id exists in the datastore."""
         pass
 
+    @staticmethod
+    def export_data(user_id):
+        """This method should be implemented in the inherited classes.
```

Comment:
in the inherited classes --> by subclasses

Reply:
Done